MAUI/ImageEditor/liquid-glass-effect.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: Liquid Glass Support for .NET MAUI Image Editor Control | Syncfusion®
+description: Learn how to enable liquid glass support for the Syncfusion® .NET MAUI ImageEditor(SfImageEditor) control using SfGlassEffectsView.
```

Comment:
Learn how to enable and customize the Liquid Glass Effect in the Syncfusion® .NET MAUI Image Editor (SfImageEditor) control.